managedplugin/download.go

```diff
@@ -128,6 +128,9 @@ type HubDownloadOptions struct {
 }
 
 func DownloadPluginFromHub(ctx context.Context, ops HubDownloadOptions) error {
+	if ops.AuthToken == "" {
+		return fmt.Errorf("an authentication token is required to download plugins from the CloudQuery registry. Try logging in via `cloudquery login`")
```

Comment:
I would make this an exported error (e.g. `ErrRequiresLogin`) and remove the "Try logging in" part. Which can be added via the CLI later, after checking with `errors.Is`?

Reply:
Good suggestion, extracted in https://github.com/cloudquery/plugin-pb-go/pull/200/commits/42aed399a8cc95407cc9a889b8d8fed45897f0e9

Reply:
We probably would want to add more errors from https://github.com/cloudquery/plugin-pb-go/blob/f5e58744d876169e3761755ca38ab0462f0228a6/managedplugin/download.go#L144 in a separate PR